README.md

```diff
@@ -83,6 +83,20 @@ docker push gcr.io/${GOOGLE_CLOUD_PROJECT}/gcsfuse
 2. Files other than those directly mentioned by the `--inputs` flag will be
 available to containers, since the entire bucket is mounted.
 
+### SSH into the worker machine
+
+The pipelines tool can start a docker container with a ssh server on the same 
+machine as the worker, to allow you to access logs
```

Comment:
The pipelines tool can start an ssh container in the background to allow you to log in using SSH and view logs in real time.